app/views/layouts/application.html.slim

```diff
@@ -19,12 +19,20 @@ html
     = render "shared/header"
 
     - unless flash.empty?
-      article.alert.row.ten.columns.offset-by-one
+      article.flash.row.ten.columns.offset-by-one
+        /* below comment is the Devise canonical way to do alerts and notices */
+        /* p.notice */
+        /*   = notice */
+        /* p.alert */
+        /*   = alert */
         ul
           - flash.each do |msg_type, msg|
-            .alert class="alert-#{msg_type}"
-              li #{msg_type}: #{msg}
+            li
+              = content_tag :span, ((Rails.env.test? || Rails.env.development?) ? "#{msg_type}: " + msg : msg), class: "flash #{msg_type}"
 
     = yield
 
     = render "shared/footer"
+    - if Rails.env.development?
```

Comment:
I probably wouldn't leave this in the finished project, even if it is environment-gated to development.